draft-ietf-tls-tls13.md

```diff
@@ -3056,11 +3086,16 @@ ticket_age_add
   the client.
 {:br }
 
+Implementations that receive a NewSessionTicket without having advertised
```

Comment:
Thinking about consistency amongst the various paragraphis similar to this one; should this one specify clients as receiving NST?